cogs/assistance-cmds/tinfoil.switch.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Why you shouldn't use tinfoil
```

Comment:
Capitalize Tinfoil, it is a proper noun

Fix this in every other instance of Tinfoil